src/sysupdate/sysupdate.c

```diff
@@ -1543,7 +1543,7 @@ static int run(int argc, char *argv[]) {
                 return r;
 
         /* SIGCHLD signal must be blocked for sd_event_add_child to work */
-        BLOCK_SIGNALS(SIGCHLD);
+        assert_se(sigprocmask_many(SIG_BLOCK, NULL, SIGCHLD) >= 0);
```

Comment:
Is this the one about using `sigprocmask` instead of `sigprocmask_many`?

Reply:
Yep.

Reply:
I did change to `sigprocmask`, then changed it back in another round of review after being recommended to use `sigprocmask_many` instead.  Personally I think using `sigprocmask_many` is preferable because there's less boilerplate, but I'm happy to change it back to `sigprocmask` again.

Reply:
If they had you go back and forth, it's clearly not all that important